src/Queue/QueueService.php

```diff
@@ -86,6 +87,7 @@ protected function createQueueTableIfNotExisting(\Closure $callable = null)
         $this->getDb()->executeQuery(sprintf('CREATE TABLE IF NOT EXISTS %s (
             id bigint AUTO_INCREMENT,
             timestamp bigint NULL,
+            userOwner int unsigned NOT NULL DEFAULT \'0\',
```

Comment:
```suggestion
            userOwner int unsigned NOT NULL DEFAULT 0,

```

I think we do not need the escaping here for 0 right?